signup.html

```diff
@@ -1,54 +1,83 @@
 <html>
-    <head>
-        <link rel="stylesheet" type="text/css" href="style.css" /> 
-        <title>EMS- Signup</title>
-        <link href="https://unpkg.com/tailwindcss@^1.0/dist/tailwind.min.css" rel="stylesheet">
-    </head>
-    <body>
-    <div class="nav h-16">
-    <a href="index.html">EMPLOYEE MANAGEMENT SYSTEM  </a>  
+  <head>
+    <link rel="stylesheet" type="text/css" href="style.css" />
+    <title>EMS- Signup</title>
+    <script src="https://cdn.tailwindcss.com"></script>
+    <style type="text/tailwindcss">
+      @layer utilities {
+  a {
+    color: rgb(193, 255, 255);
+  }
+  body {
+    background-color: rgb(177, 185, 192);
+  }
+  label {
+    color: rgb(180, 180, 180);
+    font-size: large;
+  }
+  input {
+    border: none;
+    padding: 4px 10px;
+    width: 300px;
+    font-size: large;
+  }
+  .error {
+    color: rgb(250, 134, 134);
+  }
+}
+    </style>
```

Comment:
Pls create separate CSS file for signup page

Reply:
fixed